SonarQube.TeamBuild.Integration/TeamBuildSettings.cs

```diff
@@ -94,11 +99,14 @@ public static TeamBuildSettings GetSettingsFromEnvironment(ILogger logger)
                     settings = new TeamBuildSettings()
                     {
                         BuildEnvironment = env,
+                        SourcesDirectory = Directory.GetParent(Directory.GetCurrentDirectory()).FullName
```

Comment:
I verified all 3 cases. The sources directory is indeed one up from the current dir for he cmd line scenario.